.circleci/config.yml

```diff
@@ -187,6 +187,7 @@ jobs:
       - run:
           name: Build apps
           command: yarn build:webpack --env.buildtype vagovprod --env.scaffold
+          no_output_timeout: 15m
```

Comment:
Curious: What was the motivation reducing the time from `20m` to `15m`?

Reply:
20 minutes was a bit excessive, but I added the setting when the tests were unpredictable, and I wasn't sure how long the longest run would take.

The build is more predictable, and it's rarely gone over 10 minutes.